contrib/util/language_translations/currentConstants.txt

```diff
@@ -8440,7 +8440,7 @@ Review and Approve
 Review Audits
 Review of PMH:
 Review of System
-Review Of Systems
+Review of Systems
```

Comment:
Hi @TheYairDa ,

This file is created in the translation pipeline and is not meant to be manually edited:
https://github.com/openemr/openemr/blob/master/contrib/util/language_translations/README